dev_setup.sh

```diff
@@ -74,8 +78,20 @@ for var in "$@" ; do
     if [[ ${var} == "-fm" ]] ; then
         opt_forcemimicbuild=true
     fi
+
+    if [[ ${prev} == "--prefix" ]]; then 
+        prefix=${var}
+    fi
+
+    prev=${var}
 done
 
+echo ${prefix}
+pfound=$(grep "^\.prefix$" .gitignore | wc -l)
+if [[ $pfound != 0 ]]; then 
+	echo ".prefix" >> .gitignore; 
+fi
+
```

Comment:
.gitignore doesn't need to be updated by the dev_setup script. You can just add .prefix to the .gitignore file and commit that one.

This might even cause some issues since the .gitignore is under version control and after the dev_setup script the repo would read as dirty.